app/controllers/application_controller.rb

```diff
@@ -124,6 +124,11 @@ def respond_not_found(msg = 'Not Found')
     respond_with_error(msg, 404)
   end
 
+  def respond_not_found_with_auth_bypass(msg = 'Not Found')
+    authorize! :read, nil
```

Comment:
I remember that we concluded `skip_authorization_check` shouldn't be used  as a before action but I have no idea why it wouldn't work when called explicitly.